js_tutorial/exercises/004_palindrome.js

```diff
@@ -0,0 +1,34 @@
+//4. Enter a  string and find if it's a palindrome
+
+const string = "malayalam";
+const palindrome = (str) => {
+    let reverse = str.split("").reverse().join("");   //using join("") to eliminate commas while joining the string
+    if (reverse === string) {
+        console.log(`The string "${string}" is a palindrome`);
+    }
+    else {
+        console.log(`The string "${string}" is not a palindrome`);
+    }
+}
+palindrome(string);
```

Comment:
Try reading the input parameter string from the command line.

Reply:
We have been instructed to not use deadline function to input strings